test/System/Time.cpp

```diff
@@ -3,6 +3,17 @@
 
 #include <doctest.h>
 
+using namespace std::chrono_literals;
+
+namespace std
+{
+template <typename Rep, typename Period>
+std::ostream& operator<<(std::ostream& out, const std::chrono::duration<Rep, Period>& duration)
+{
+    return out << std::chrono::microseconds(duration).count() << "us";
+}
```

Comment:
I am not sure why, but by trying it I find that if we include <doctest.h> after declaring (or defining) the output operator, putting it in namespace std is not required.

Reply:
Apparently this is some C++ weirdness related to argument-dependent lookup.
This paper gives details. https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p0934r0.pdf
In particular section 3.4 describes a similar situation, and section 4.1 explains why a global `operator<<` is generally not a good idea.
To my understanding, it is better as it is now in this PR.

Reply:
The third stringification alternative doctest supports seems to have none of the drawbacks we noticed.
```c++
namespace doctest
{
    template <typename Rep, typename Period>
    struct StringMaker<std::chrono::duration<Rep, Period>>
    {
        static String convert(const std::chrono::duration<Rep, Period>& duration)
        {
            return toString(std::chrono::microseconds(duration).count()) + "us";
        }
    };
}
```

Reply:
Oh yeah idk why I looked past that. I'll let @Bromeon decide if we care about avoiding `namespace std`.

Reply:
> The third stringification alternative doctest supports seems to have none of the drawbacks we noticed.

If doctest provides a way that works without drawbacks, should we go for this one, just to be on the safe side? Maybe along with a comment why we don't use `operator<<` like everywhere else.

But I can definitely also live with the `std` solution, if you find that nicer. I would add a comment in that case as well.

Reply:
I went ahead and used the option that doesn't requiring opening `namespace std`. I went ahead and did the same thing for the `std::vector` printer in test/Graphics/Transform.cpp since I'm not positive that that's legal either.